frb_codegen/src/lib.rs

```diff
@@ -64,13 +64,10 @@ pub fn frb_codegen_multi(
     info!("Picked config: {:?}", config);
 
     info!("Phase: Parse source code to AST, then to IR");
-    let raw_ir_file = config.get_ir_file()?;
-
-    info!("Phase: Transform IR");
-    let ir_file = transformer::transform(raw_ir_file);
```

Comment:
nit: curious why we move the `transform` phase into other places?

Reply:
I didn't want to  move the `transformer::transform` initially, but finally, it turns out that even the shared `ir_file` needs it. So I moved it into `get_ir_file(...)`. And what you said does let me realize a little lint related in [b80f361](https://github.com/fzyzcjy/flutter_rust_bridge/pull/1145/commits/b80f3613de66609741e83309201c78df96547cef).